django_ticketoffice/decorators.py

```diff
@@ -30,7 +30,7 @@ def guest_login(request, invitation):
         setattr(request, 'cache', {'invitation': invitation})
     request.user = GuestUser(invitation=invitation, invitation_valid=True)
     try:
-        request.user.id = invitation.data['user']
+        request.user.id = (invitation.data or {})['user']
```

Comment:
This is a bit cryptic, why not also expect TypeError below?

Reply:
Let's do this!